spec/components/commons/objects.yaml

```diff
@@ -33,6 +33,18 @@ components:
           $ref: "fields.yaml#/components/schemas/Duration"
         unit:
           $ref: "fields.yaml#/components/schemas/DurationUnit"
+    Special:
+      type: object
+      properties:
+        original_price:
+          $ref: "../commons/fields.yaml#/components/schemas/PriceField"
+        savings:
+          type: number
+          description: The maximum possible discount in percent.
+          example: 12.0
+        apps_only:
```

Comment:
I think this is not used. Is there something forseeable in the future where we might need this field or can we remove it?

Reply:
I think we should keep it. This info is there to display possible discounts on the tours or tour_option level.